### ISSUE TITLE
Support for (Neo)Vim?

### ISSUE THREAD

**Issue by gzagatti** [fniessen/emacs-leuven-theme#94]
I really like this theme on Emacs.

I am wondering if there is any plan to port it to (Neo)vim or you know of any work in progress in this direction?

**Comment by fniessen** [comment]
Dear @gzagatti ,

Thanks for the comments!

And, no... I'm devoted to Emacs only -- and am not aware of anybody trying to port my Leuven-theme to another platform. Could exist, though, and would be welcome...

Would you give it a try?

**Comment by gzagatti** [comment]
I will give it a try within the next couple of weeks. Thanks for the quick response.

**Comment by gzagatti** [comment]
@fniessen I have created the vim port of the Leuven theme. It is available in [gzagatti/vim-leuven-theme](https://github.com/gzagatti/vim-leuven-theme). It would be great if it could be advertised in your page for those looking for the same theme in Vim.

I have tried to keep the scheme as close as possible to the original, while keeping in mind the constraints imposed by Vim. Feel free to provide feedback.
